lib/nexmo_developer/app/presenters/page_title.rb

```diff
@@ -1,18 +1,21 @@
 class PageTitle
-  DEFAULT = 'Vonage API Developer'.freeze
-
   def initialize(product, document_title)
     @product        = product
     @document_title = document_title
   end
 
   def title
     if @product && @document_title
-      "#{@product.titleize} > #{@document_title} | #{DEFAULT}"
+      "#{@product.titleize} > #{@document_title} | #{default_title}"
     elsif @document_title
-      "#{@document_title} | #{DEFAULT}"
+      "#{@document_title} | #{default_title}"
     else
-      DEFAULT
+      default_title
     end
   end
+
+  def default_title
+    config = YAML.safe_load(File.open("#{Rails.configuration.docs_base_path}/config/header_meta.yml"))
```

Comment:
@benhayehudi  I can't make the indention to work in a suggestion :(
but can you memoize the value here?

```
  @default_title ||= begin
    config = YAML.safe_load(File.open("#{Rails.configuration.docs_base_path}/config/header_meta.yml"))
    config['title']
  end
```

Reply:
done 👍